Commands.adoc

```diff
@@ -0,0 +1,226 @@
+= Available Commands and Examples
+:toc:
+:toc-placement!:
+
+Once you do `vagrant up` and run an ADB box, as a developer you may want to
+customize or set up your development environment to optimally utilize the ADB as
+ per your requirements.
+
+The vagrant-service-manager plugin enables you to check and tweak your
+container development environment.
+
+'''
+toc::[]
+'''
+
+You can use the vagrant service manager to set up your environment variables and
+ get the TLS certificates; identify the routable ip network and version of your
+ box; and check the status and manage the lifecycle of the configured services
+ in the running VM.
+
+To run the vagrant-service-manager plugin use: +
+-------------------------------------------
+vagrant service-manager [command] [options]
+-------------------------------------------
+The following commands can be used: +
+`env`: Displays the connection information for services in the ADB box +
+`box`: Displays the box related information like version, release, IP etc +
+`status`: Lists services and their running state +
+`start`: Starts the given service in the box +
+`stop`: Stops the given service in the box +
+`restart`: Restarts the given service in the box +
+
+The following options can be used: +
+`--script-readable`: Displays information in a key=value format +
+`-h, --help, print this help`: Prints help for the specific command being run +
+
+== Get the Environment Variables and Certificates
+
+If you need to get the environment variables and certificates for all the active
+ services in the box, run: +
+
+`$ vagrant service-manager env` +
+
+Usage: `vagrant service-manager env [service] [options]`
+
+The possible options for service are Docker and OpenShift.
+If no service is specified `vagrant service-manager env` will run the plugin and
+ provide connection information for all the active services .
+
+The following options can be used: +
+`--script-readable` : Displays information in a key=value format +
+`-h, --help, print this help` : Prints help for the specific command being run. +
+
+
+=== Examples
+. If you need connection information for all active services in the box in a
+manner that can be evaluated in a shell.
++
+--
+-----------------------------------------------------------------------------
+   $ vagrant service-manager env
+
+   # docker env:
+   # Set the following environment variables to enable access to the
+   # docker daemon running inside of the vagrant virtual machine:
+   export DOCKER_HOST=tcp://10.1.2.2:2376
+   export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
+   export DOCKER_TLS_VERIFY=1
+   export DOCKER_API_VERSION=1.21
+
+   # openshift env:
+   # You can access the OpenShift console on: https://10.1.2.2:8443/console
+   # To use OpenShift CLI, run: oc login https://10.1.2.2:8443
+   export OPENSHIFT_URL=https://10.1.2.2:8443
+   export OPENSHIFT_WEB_CONSOLE=https://10.1.2.2:8443/console
+
```

Comment:
currently we have empty lines though. @budhrg do remember where we stand with this. I just wish we never did this 'env' command with no argument :-(